core/logger/logger.go

```diff
@@ -187,19 +190,25 @@ func NewLogger() Logger {
 }
 
 type Config struct {
-	LogLevel    zapcore.Level
-	Dir         string
-	JsonConsole bool
-	ToDisk      bool // if false, the Logger will only log to stdout.
-	UnixTS      bool
+	LogLevel                   zapcore.Level
+	Dir                        string
+	JsonConsole                bool
+	UnixTS                     bool
+	ToDisk                     bool // if false, the Logger will only log to stdout.
```

Comment:
How does ToDisk play with the new debug logging?

Reply:
I think I've have misinterpreted what was needed here. I thought the logging to disk flag would've had some effect in this. but now that I think about it, it shouldn't? 

cause we want the logs anyways...